README.md

```diff
@@ -2384,11 +2384,11 @@ import kyo.*
 
 // An example computation with fibers
 val a: Int < Async =
-    Fiber.init(Math.cos(42).toInt).map(_.get)
+    Fiber.initUnscoped(Math.cos(42).toInt).map(_.get)
 
 // Avoid handling 'Async' directly
 val b: Fiber[Int, Any] < Sync =
-    Fiber.init(a)
+    Fiber.initUnscoped(a)
```

Comment:
@johnhungerford `Fiber.init` ? There are a lot of cases where it should be `Fiber.init`

Reply:
+1 I think we should prefer the scoped versions in the readme